.changes/unreleased/NOTES-20240216-090121.yaml

```diff
@@ -0,0 +1,8 @@
+kind: NOTES
+body: 'function: Changed the framework type for variadic parameters to `types.TupleType`,
+  where each element is the same element type. Provider-defined functions using a
+  `types.List` for retrieving variadic argument data will need to update their code
+  to use `types.Tuple`.'
```

Comment:
Open to suggestions on wording here. I chose a `NOTES` update because the provider-defined functions are still in tech-preview.

Reply:
My personal opinion is that it should either be in breaking changes or it can be in notes if it mentions the feature its still in technical preview. I would much rather get 👀 on the "scary" change (and hey maybe get more folks looking at it) rather than trying to bury the lede.